scripts-dev/lint.sh

```diff
@@ -7,7 +7,15 @@
 
 set -e
 
-isort -y -rc synapse tests scripts-dev scripts
-flake8 synapse tests
-python3 -m black synapse tests scripts-dev scripts
+if [ $# -ge 1 ]
+then
+  files=$*
+else
+  files="synapse tests scripts-dev scripts"
```

Comment:
Note that this will have flake8 run on scripts-dev and scripts. I don't think that's a problem, but someone may have some concerns.

Reply:
Right; I meant to note that in the PR summary.